lib/plugins/pseudo-levels/index.js

```diff
@@ -108,14 +108,15 @@ function getSymlinks(root, requiredTargets, fileSuffixes) {
     return Object.keys(symlinks);
 }
 
-function registerNodes(magic, root, filenames) {
+function registerNodes(magic, root, filenames, destPath) {
     var nodes = {};
 
     filenames.forEach(function (filename) {
         var target = path.relative(root, filename),
             node = path.dirname(target);
```

Comment:
I think that need remove `target` variable and build relative path from `dstpath`:

```js
var dstpath = path.join(root, destPath),
    node = path.relative(dstpath, path.dirname(filename));
```

Then the condition below is not necessary to change.